src/modules/85-cv/components/CommonMultiItemsSideNav/components/CommonSelectedAppsSideNav/components/GroupedSideNav/components/MetricMenu.tsx

```diff
@@ -69,39 +69,41 @@ export default function MetricMenu({
   }, [index, itemName, showPromptOnDelete])
 
   return (
-    <Popover
-      isOpen={popoverOpen}
-      interactionKind={PopoverInteractionKind.HOVER}
-      className={Classes.DARK}
-      position={Position.RIGHT_TOP}
-      onInteraction={nextOpenState => {
-        setPopoverOpen(nextOpenState)
-      }}
-    >
-      <Icon
-        data-testid={`sideNav-options`}
-        name="Options"
-        onClick={() => {
-          setPopoverOpen(!popoverOpen)
+    <>
+      <Popover
+        isOpen={popoverOpen}
+        interactionKind={PopoverInteractionKind.HOVER}
+        className={Classes.DARK}
+        position={Position.RIGHT_TOP}
+        onInteraction={nextOpenState => {
+          setPopoverOpen(nextOpenState)
         }}
-      />
-      <Menu>
-        <Menu.Item
-          data-testid={`sideNav-edit`}
-          icon="edit"
-          text={editLabel ?? <String stringID="edit" />}
-          onClick={onEdit}
-        />
-        <Menu.Item
-          data-testid={`sideNav-delete`}
-          icon="trash"
-          text={deleteLabel ?? <String stringID="delete" />}
-          onClick={(e: React.MouseEvent) => {
-            e.stopPropagation()
-            openDialog()
+      >
+        <Icon
+          data-testid={`sideNav-options`}
+          name="Options"
+          onClick={() => {
+            setPopoverOpen(!popoverOpen)
           }}
         />
-      </Menu>
-    </Popover>
+        <Menu>
+          <Menu.Item
+            data-testid={`sideNav-edit`}
```

Comment:
data-testid={`sideNav-edit`}
```suggestion
            data-testid='sideNav-edit'
```

Reply:
done.